servicemesh-install/topics/install-post.adoc

```diff
@@ -1,13 +1,25 @@
 [[verifying-installation]]
 = Verifying the installation
 
-After the `openshift-ansible-istio-installer-job` has completed, run the following command:
+Run the following command to determine if the operator finished deploying the control plane.
+
+----
+$ oc get controlplane/istio-installation -n istio-system --template='{{range .status.conditions}}{{printf "%s=%s, reason=%s, message=%s\n\n" .type .status .reason .message}}{{end}}'
```

Comment:
We might want a comment here saying that istio-installation should be the name of the resource

Reply:
I added a note stating: "The name of the resource is istio-installation."

Reply:
Added a note.